phoenicis-repository/src/main/java/org/phoenicis/repository/RepositoryManager.java

```diff
@@ -42,6 +42,14 @@
      */
     ScriptDTO getScript(List<String> path);
 
+    /**
+     * This method returns the {@link org.phoenicis.repository.dto.ScriptDTO} with the given ID
+     *
+     * @param id The script ID
+     * @return The found ScriptDTO
+     */
+    ScriptDTO getScript(String id);
```

Comment:
Do you plan to remove the `ScriptDTO getScript(List<String> path);` method long-term?

Reply:
That's the idea. However, it's not a simple change so I didn't want to do it now.

Reply:
In the same class. The idea is just to give a explainatory name to the pattern

Reply:
@qparis in this case we would still have the issue that the pattern is repeated in several classes.